### PR TITLE
feat: add props for enhanced customization

### LINKED ISSUE #55: Add configuration props
**Describe the bug**
Add more props for customization. ex: filterSide -> left | right, and filterOpen -> true | false

### LINKED ISSUE #57: Add Prop to auto-expand most recent response
**Is your feature request related to a problem? Please describe.**
When implemented in inspector it would be nice if the most recent response could have it's payload auto-expanded.

**Describe the solution you'd like**
Add a prop that allows for the expansion of the most recent response payload

### PR DESCRIPTION
**Pull request by mattbford**
Adds three props. 
sidebarOpen: whether or not the sidebar is open by default
sidebarAlign: which side of the screen is the sidebar on, Right or Left
openRecentPayload: if true, expands most recent response payload automatically.

fixes #57
fixes #55

### CODE REVIEW

File: src/components/logsReact/logsReact.tsx [hunk 1]
```diff
@@ -78,28 +97,66 @@ const JSONRPCLogger: React.FC<IProps> = (props) => {
...
-      <div className={clsx(classes.content, {
-        [classes.contentShift]: drawerOpen,
-      })}>
-        <CardList logs={props.logs} filter={methodFilter} />
-      </div>
+    <div>
+      { sidebarAlignment === "left" ?
+        <div className={classes.logsReact}>
+          <div className={clsx(classes.extendDiv, drawerOpen && classes.hide)}>
+            <IconButton
```
Comment (meowsbits) [inline src/components/logsReact/logsReact.tsx:104]: Why are these respective left/right child elements in different orders?

Comment (mattbford) [reply src/components/logsReact/logsReact.tsx:104]: To position the sidebar div correctly. For example, if the code for sidebarAlignment "right" was the same as "left" then the sidebar would slide in from the right but still appear on the leftside.

Comment (meowsbits) [reply src/components/logsReact/logsReact.tsx:104]: Gotcha. Thanks. LGTM but what do I know ;)

**Pull request merged**